app/models/manageiq/providers/vmware/infra_manager.rb

```diff
@@ -337,7 +337,7 @@ def self.provision_class(via)
       end
     end
 
-    def self.default_blacklisted_event_names
+    def self.filtered_event_names
       Settings.ems.ems_vmware.blacklisted_event_names
     end
```

Comment:
Minor, but we might want a comment here explaining why we aren't using the base class method.